src/path_functions.cpp

```diff
@@ -173,3 +131,44 @@ QString AOApplication::get_case_sensitive_path(QString p_file)
   return file_parent_dir + "/" + file_basename;
 }
 #endif
+
+QString AOApplication::find_asset_path(QStringList possible_paths)
+{
+  return find_asset_path(possible_paths, {""});
```

Comment:
...Was this supposed to be an attempt at default arguments?

I believe you can just do `QString AOApplication::find_asset_path(QStringList possible_paths, QStringList possible_exts = {""})` during function definition.

Reply:
Yes, can fix